### PR TITLE
Add log messages GetRange request 

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**
It's helpful to add info/trace messages to ease the debugging of
the GetRange request.

### CODE REVIEW

File: src/dlsproto/client/request/internal/GetRange.d [hunk 1]
```diff
@@ -588,6 +603,8 @@ private scope class GetRangeHandler
         {
             while (this.waitForRecords())
             {
+                .log.trace("[{}]:[{}][RecordStream] Got a batch of records, stopped is {}.",
```
Comment (gavin-norman-sociomantic) [inline src/dlsproto/client/request/internal/GetRange.d]: What's the default logging level?

**Review by gavin-norman-sociomantic — COMMENTED** [review]
I'm a bit on the edge about this. It's debug info, not info that is generally of interest to users of the DLS client (i.e. it refers to internals of the GetRange implementation, rather than the public API of the GetRange request). What do you think?

**Comment by nemanja-boric-sociomantic** [comment]
I was also a bit on the edge here, but I found the precedent in dhtproto's ChannelMirror :-) (and this entire thing was inspired by it).

File: src/dlsproto/client/request/internal/GetRange.d (hunk 1 above)
Comment (nemanja-boric-sociomantic) [reply src/dlsproto/client/request/internal/GetRange.d]: Depens on how the application configures the loggers. I don't think they default to `trace`, though.

**Comment by gavin-norman-sociomantic** [comment]
I was going to say that the channel mirror logging is more on the level of API-features, but actually it isn't really. It's largely about the internal implementation (a bundled GetAll and Listen request).

**Comment by nemanja-boric-sociomantic** [comment]
I think that as well. I think it's not that hard to remove this info if you don't need it, and even when that's not done, using the `info` level, this should not be spam the application log, but it should reflect the actions of the user.

**Comment by nemanja-boric-sociomantic** [comment]
Testing this in production shows that the `trace` level is too chatty - it seems that `Got a batch of records` log event generates gigabytes of data in half of hour, so I'll have to shrink that down.

**Comment by nemanja-boric-sociomantic** [comment]
I don't think we're going to move forward with this one. Closing.

**Pull request closed without merging**